pom.xml

```diff
@@ -39,7 +39,8 @@
   </developers>
 
   <properties>
-     <jenkins.version>1.625.3</jenkins.version>
+     <jenkins.version>2.7.1</jenkins.version>
```

Comment:
I'll have to take a closer look again. I think there is a way to allow it to work even with older jenkins version.